scripts/genes_MAGS_eggNOG_mapping.py

```diff
@@ -121,7 +120,7 @@ def load_mags_contigs_taxonomies_for_sample(sample_dir, taxonomy_path, checkm_pa
     colnames = ["Bin Id", "# genomes","# markers", 
                 "Completeness", "Contamination", "Strain heterogeneity"]
     checkm_df = pd.DataFrame(columns=colnames)
-    for file in glob.glob(os.path.join(checkm_path, f"{mag_root}*.txt")):
+    for file in glob.glob(os.path.join(checkm_path, f"{mag_root}_checkm.txt")):
```

Comment:
Is this for loop necessary? There is only in file per sampleID/mag_root as far as I understand.